flash/vision/detection/data.py

```diff
@@ -23,10 +23,11 @@
 from torch.utils.data._utils.collate import default_collate
 from torchvision import transforms as T
 
-from flash.core.data import TaskDataPipeline
-from flash.core.data.datamodule import DataModule
-from flash.core.data.utils import _contains_any_tensor
-from flash.vision.classification.data import _pil_loader
+from flash.data.auto_dataset import AutoDataset
+from flash.data.data_module import DataModule
+from flash.data.process import Preprocess
+from flash.data.utils import _contains_any_tensor
+from flash.vision.utils import pil_loader
 
 _COCO_AVAILABLE = _module_available("pycocotools")
```

Comment:
let's move all these to `flash/utils/imports`

Reply:
Done !